careerfairsystems/arkad-app
# Hosts added

## Change description

Companies will be able to see career fair host contact information in help. pictures added for most of them.

## How to verify

Unfortunately we are still waiting for a test account. But the functionality should work, the important thing is that it doesn't crash the app and it's compatible with persistlogin and switching from company to studentlogin.

If one wants to test it, one can log in with alexanderlundst on Nexpo and use the website or description fields that we can edit there, and replace phone/email with those fields to fake actual data (this.props.representative.company.website )

## Issues fixed

#211
